model/qti/ImportService.php

```diff
@@ -445,7 +461,8 @@ public function importQtiItem(
         $enableMetadataValidators = true,
         $itemMustExist = false,
         $itemMustBeOverwritten = false,
-        &$overwrittenItems = []
+        &$overwrittenItems = [],
+        $metaMedataValues = []
```

Comment:
You are taking all metadata from all package into `importQtiItem`. 
`importQtiItem` is importing particular item, no?